packages/strapi-design-system/src/Breadcrumbs/Breadcrumbs.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { After } from '@strapi/icons';
+import { Text } from '../Text';
+import { Box } from '../Box';
+import { useId } from '../helpers/useId';
+
+const CrumbWrapper = styled.li`
```

Comment:
```suggestion
const CrumbWrapper = styled(Row)`
```

so that you benefit from the inline prop and the align-items by default and then you can eventually do `<CrumbWrapper as="li"`